src/app/components/client/dialog/Dialog.tsx

```diff
@@ -14,6 +14,7 @@ export type Props = {
   title?: ReactNode;
   illustration?: ReactNode;
   variant?: "vertical" | "horizontal";
+  onKeyDown?: (event: KeyboardEvent) => void;
```

Comment:
This appears to be unused? (And hopefully unnecessary?)

```suggestion
```

Reply:
Yup, not needed — same as https://github.com/mozilla/blurts-server/pull/3823#discussion_r1431746339.